src/Core/Transform/ScaleAnImage/Code.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+
+# =========================================================================
+#
+#   Copyright NumFOCUS
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#          http://www.apache.org/licenses/LICENSE-2.0.txt
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# =========================================================================*/
+
+import sys
+
+import itk
+
+if len(sys.argv) < 3:
+    raise Exception(f"Usage: {sys.argv[0]} inputFile outputFile")
+
+image = itk.imread(sys.argv[1], itk.UC)
+
+Dimension = image.GetImageDimension()
+
+transform = itk.ScaleTransform[itk.D, 2].New(
+    Scale=[1.5] * Dimension, Center=[x / 2 for x in itk.size(image)]
```

Comment:
In C++, center is explicitly `float`. Here it might be implicitly `double`.

Reply:
Does changing `x / 2` to `x / 2.` address the issue?

Reply:
Unfortunately casting with `x / 2.` or `float(x / 2)` does not change the image output. I also tried explicitly setting `center` to `double` in the cxx test but it didn't seem to change the cxx image output either. In both cases the cxx baseline comparison test continues to succeed while the python baseline comparison test fails.

Reply:
Updating to read in input data rather than generate input in C++ and Python tests respectively seems to have resolved the issue.